### DISCUSSION TITLE
Syntax highlighting for Julia

### DISCUSSION THREAD

**Discussion by colbec** [dail8859/NotepadNext#464] [Q&A]
Currently NotepadNext does not have syntax highlighting for Julia.
1: Is there an alternative existing facility that closely resembles that required for Julia? I am currently using C# which helps.
2: What process would be required to add in a specific highlighter for Julia that could be edited to suit?

**Comment by dail8859** [comment]
The library used for the syntax highlighting (Lexilla) does support Julia...it just hasn't been configured at all within Notepad Next. So it's just a matter of creating a new lua config for Julia to set the colors, keywords, etc.
